canonical/ubuntu.com
# k8s-1.31

## Done

- Added components, release-notes and upgrading for 1.31
- Updated Cilium pages
- Updated supported versions
- Updated main release notes

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- NOTE that the content has already been reviewed
